pxr/imaging/hdSt/renderDelegate.cpp

```diff
@@ -205,7 +219,13 @@ HdStRenderDelegate::HdStRenderDelegate(HdRenderSettingsMap const& settingsMap)
         HdRenderSettingDescriptor{
             "Dome light camera visibility",
             HdRenderSettingsTokens->domeLightCameraVisibility,
-            VtValue(true) }
+            VtValue(true) },
+#ifdef PXR_MATERIALX_SUPPORT_ENABLED
+        HdRenderSettingDescriptor{
+            "Path to the directory of the persistent MaterialX codegen cache",
+            HdStRenderSettingsTokens->hdstMtlxCodegenCacheDirPath,
+            VtValue(HdSt_MaterialXShaderRegistry::GetCacheDirPathEnvSetting()) }
```

Comment:
The cache directory exposed as a Storm render setting. The env var supplies the default value.